src/main/resources/resources.properties

```diff
@@ -56,64 +56,64 @@ presentation.dependencyStatus.depError=Error in dependencies
 ui.courseProject.dialogs.showErrorDialog.malformedCourseConfiguration.title=Course Configuration Error
-ui.courseProject.dialogs.showErrorDialog.malformedCourseConfiguration.message=An error occurred while reading the course configuration. Please contact the course staff.
+ui.courseProject.dialogs.showErrorDialog.malformedCourseConfiguration.message=The A+ Courses plugin could not read the course configuration, perhaps due to a mistake by the course staff.\nIf the problem persists, please contact the staff.
 
-ui.courseProject.form.textTemplate=<html><body>The currently opened project will be turned into a project for <b>{0}</b>.</body><html>
-ui.courseProject.form.infoText=<html><body>The A+ Courses plugin will adjust IntelliJ IDEA settings. This helps use IDEA for coursework.</body></html>
+ui.courseProject.form.textTemplate=<html><body>This project will be turned into a project for <b>{0}</b>.</body><html>
+ui.courseProject.form.infoText=<html><body>The A+ Courses plugin will now adjust IntelliJ''s settings. This helps you use IntelliJ for coursework.</body></html>
 ui.courseProject.form.settingsOptOut=Leave IntelliJ settings unchanged.
 ui.courseProject.form.currentSettingsText=<html><body>IntelliJ IDEA settings are already imported for <b>{0}</b>.</body><html>
-ui.courseProject.form.warningText=<html>(<b>Not recommended</b>. Only pick this option if you are sure you know what you are doing.)</html>
-ui.courseProject.view=Turn Project Into A+ Course Project
+ui.courseProject.form.warningText=<html>(Leaving the settings unchanged <b>not recommended</b>. Select this box only if you know what you are doing.)</html>
+ui.courseProject.view=Turn Project Into A+ Project
 
 ## UI other
 ### Authentication
 ui.authenticationView.name=A+ Token
-ui.authenticationView.setToken=Please enter your A+ token:
-ui.authenticationView.noEmptyToken=Token must not be empty
+ui.authenticationView.setToken=Please enter your A+ access token:
+ui.authenticationView.noEmptyToken=The token cannot be empty
 
 ### Misc
-ui.aboutDialog.description=This plugin supports the educational use of IntelliJ (and its Scala plugin) in\nprogramming courses that rely on the A+ course platform, which has been\ndeveloped at Aalto University. The plugin accesses programming assignments\nand automated grading services provided by A+ and otherwise enhances\nthe student experience.
-ui.aboutDialog.title=About A+ Plugin
+ui.aboutDialog.description=The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin) in\nprogramming courses that rely on the A+ learning management system, which has been\ndeveloped at Aalto University. The plugin accesses programming assignments\nand automated grading services provided by A+ and otherwise enhances\nthe student experience. Web site: https://plugins.jetbrains.com/plugin/13634-a-courses
```

Comment:
@Aalto-LeTech/idea-plugin-dev need to remember replacing the URL with a corresponding Swing element (link `LinkLabel` so it would look properly